lib/gs1/batch.rb

```diff
@@ -0,0 +1,35 @@
+module GS1
+  # Batch or lot number: AI (10)
+  #
+  # The GS1 Application Identifier (10) indicates that the GS1 Application Identifier data field contains a
+  # batch or lot number. The batch or lot number associates an item with information the manufacturer
+  # considers relevant for traceability of the trade item to which the element string is applied. The data may
+  # refer to the trade item itself or to items contained. The number may be, for example, a production lot
+  # number, a shift number, a machine number, a time, or an internal production code. The data is
+  # alphanumeric and may include all characters contained in figure 7.11-1.
```

Comment:
Where is figure 7.11-1?

Reply:
https://www.gs1.org/sites/default/files/docs/barcodes/GS1_General_Specifications.pdf